test/kubemark/start-kubemark.sh

```diff
@@ -97,7 +138,8 @@ gcloud compute ssh --zone=${ZONE} --project="${PROJECT}" ${MASTER_NAME} \
   sudo bash -c \"echo \"${KUBE_BEARER_TOKEN},admin,admin\" > /srv/kubernetes/known_tokens.csv\" && \
   sudo bash -c \"echo \"${KUBELET_TOKEN},kubelet,kubelet\" >> /srv/kubernetes/known_tokens.csv\" && \
   sudo bash -c \"echo \"${KUBE_PROXY_TOKEN},kube_proxy,kube_proxy\" >> /srv/kubernetes/known_tokens.csv\" && \
-  sudo bash -c \"echo admin,admin,admin > /srv/kubernetes/basic_auth.csv\""
+  sudo bash -c \"echo admin,admin,admin > /srv/kubernetes/basic_auth.csv\" && \
+  sudo bash -c \"echo -n ${EVENT_STORE_IP:-127.0.0.1} > /tmp/event-store-ip\""
```

Comment:
I prefer passing it to start-kubemark-master.sh as you suggest. It's not all about basic readability - it's more about reducing coupling. https://en.wikipedia.org/wiki/Coupling_(computer_programming)